Doc/reference/import.rst

```diff
@@ -539,25 +539,26 @@ the module.
    the module.  This name is used to uniquely identify the module in
    the import system.
 
+   ``__name__`` is set to ``'__main__'`` for non-package modules run
```

Comment:
It is not set to `__main__` for any imported modules.  Instead, the `__main__` module is where the interpreter runs code initially, whether it be a script, stdin, etc.  So either drop this paragraph or change it to something like the following:

```
   The module in which the interpreter initially runs has ``__name__``
   set to "__main__".  This is where scripts run, whether read from
   a file or stdin.  In the case of a script (file), the "__main__" module
   is entirely distinct from a module loaded from the same file.  That
   module would have ``__name__`` set normally (matching the
   filename).  However, the two modules would have the same
   ``__file__``.
```

There could even be more detail about how a script can be entirely re-executed by later importing it as a module, but that

Reply:
Thanks for the clarification.